lib/Access.php

```diff
@@ -1369,12 +1369,17 @@ private function prepareSearchTerm($term) {
 		$config = \OC::$server->getConfig();
 
 		$allowEnum = $config->getAppValue('core', 'shareapi_allow_share_dialog_user_enumeration', 'yes');
+		$allowMedialSearches = $config->getSystemValue('user_ldap.medial_search.enable', false);
```

Comment:
you probably saw that there is only one other key called `ldapUserCleanupInterval`.
I'm not too happy about the naming scheme inconsistenc<.

On the other hand, the naming you used makes more sense for the future. So I think it's ok. Please add this to config.sample.php.